### PR TITLE
vale: Add version 2.10.6

### PR DESCRIPTION
**Pull request by arvikon**

### CODE REVIEW

File: bucket/vale.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "2.10.6",
+    "description": "A syntax-aware linter for prose built with speed and extensibility in mind.",
+    "homepage": "https://docs.errata.ai/vale/about/",
+    "license": "MIT",
+    "url": "https://github.com/errata-ai/vale/releases/download/v2.10.6/vale_2.10.6_Windows_64-bit.zip",
```
Comment (Ash258) [inline bucket/vale.json:6]: Clearly 64bit only

File: bucket/vale.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "hash": "5ad04b54a61d8acf9cb636329721918fca407a0d072177c9b2638e91804805ac",
+    "bin": "vale.exe",
+    "checkver": {
+        "github": "https://github.com/errata-ai/vale"
+    },
+    "autoupdate": {
+        "url": "https://github.com/errata-ai/vale/releases/download/v$version/vale_$version_Windows_64-bit.zip",
+        "hash": {
+            "url": "$baseurl/vale_$version_checksums.txt",
+            "regex": "$sha256\\s+$basename"
```
Comment (Ash258) [inline bucket/vale.json:16]: I do not beleive this is needed

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/vale.json (hunk 1 above)
Comment (arvikon) [reply bucket/vale.json]: Thx, updated  with instructions for 64-bit architecture only.

File: bucket/vale.json (hunk 2 above)
Comment (arvikon) [reply bucket/vale.json]: Makes sense. Removed regex.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**